### ISSUE TITLE
How do you get the new visualiser?

### ISSUE THREAD

**Issue by skylogic004** [spotify/luigi#1658]
The docs show a nice screenshot: https://raw.githubusercontent.com/spotify/luigi/master/doc/visualiser_front_page.png

But how do you enable it? My dashboard is still the boring white one. I'm using Luigi version 2.1.1.
![image](https://cloud.githubusercontent.com/assets/4992373/14652613/e9cddce0-0629-11e6-96ae-687263c02fa3.png)

- Someone said "The theme can be set with the visualizer_theme option as either d3, svg or lte." (https://github.com/spotify/luigi/pull/1086#issuecomment-124513250) but I don't know where to set it. Is it in config.cfg? What section should it be under?
- I tried going to http://192.168.10.99:8082/static/visualiser/index.lte.html but it says 404: Not Found.
- I tried searching the documentation for "theme" and "visualiser" (http://luigi.readthedocs.org/en/stable/search.html?q=theme&check_keywords=yes&area=default#) but nothing came up.
- Some places use "visualizer" and some places use "visualiser" which makes things confusing.

Thanks

**Comment by Tarrasch** [comment]
> Someone said "The theme can be set with the visualizer_theme option as either d3, svg or lte." (#1086 (comment)) but I don't know where to set it. Is it in config.cfg? What section should it be under?

That's old information. Does not exist anymore.

> Some places use "visualizer" and some places use "visualiser" which makes things confusing.

D'oh, do you mind sending a PR consolidating the docs? :)

----

To be honest. I think you're simply running *old* code somewhere. Make sure there's no PYTHONPATH, virtualenv, etc. etc., also remember that server and client run independently.

**Comment by skylogic004** [comment]
Yes, it looks like there was indeed old code. I had used pip to update an old version to the new version. I guess it didn't update everything. For instance, the following seemed to be old:
/usr/local/bin/luigi
/usr/local/bin/luigid

So, I uninstalled (pip uninstall luigi), removed the luigi* scripts in /usr/local/bin, cloned from the luigi git repo, and ran _python setup.py develop_. Now it works great and has the new visualiser.

Thanks for your help.
